modules/update-mirror-repository.adoc

```diff
@@ -42,14 +54,16 @@ $ LOCAL_REPOSITORY='<local_repository_name>'
 For `<local_repository_name>`, specify the name of the repository to create in your
 registry, such as `ocp4/openshift4`.
 
-.. Export the name of the repository to mirror:
+.. Export a tag name for the mirrored release image:
 +
 [source,terminal]
 ----
-$ PRODUCT_REPO='openshift-release-dev'
+$ LOCAL_TAG=<local_tag>
```

Comment:
Sounds like this VAR is not needed any more, if use digest when doing mirror operation.